portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import com.google.sps.data.Message;
+import java.util.ArrayList;
+
+/** Servlet that stores new comments. */
+@WebServlet("/new-data")
+public class NewCommentServlet extends HttpServlet {
+
+  private static final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
exceed 80 characters?
please check this for all files in this change.

Reply:
Will it be a bit weird if I divide this statement into two lines
`private static final DatastoreService datastore = 
  DatastoreServiceFactory.getDatastoreService();` 
as it is just an initialization?

Reply:
No. But you need to indent correctly so others could understand they are actually one sentence at the first glance.
```
private static final DatastoreService DATASTORE =
    DatastoreServiceFactory....

doGet(request, 
      response)
```

Reply:
Got it, thank you!